skycoin/skywire
# Add router related config to the manager

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- Now the min and max hops are shown in the right bar of the visor details page. Also, under the data there is a button for being able to change the values.

How to test this PR:
Enter to the visor details page and check the data in the right bar. For changing the values, press the `Change configuration` button.

Note: the PR will be finished after the API endpoints for getting and setting the data are added.
